frb/builds/build_hosts.py

```diff
@@ -0,0 +1,214 @@
+""" Top-level module to build or re-build the JSON files for
+FRB host galaxies"""
+
+from pkg_resources import resource_filename
+import os
+import sys
+
+from IPython import embed
+
+import numpy as np
+
+from astropy.coordinates import SkyCoord
+from astropy import units
+from astropy.table import Table
+
+from frb.galaxies import frbgalaxy, defs
+from frb.galaxies import photom as frbphotom
+from frb.surveys import des
+
+db_path = os.getenv('FRB_GDB')
+if db_path is None:
+    embed(header='You need to set $FRB_GDB')
+
+
+def build_host_121102(build_photom=False):
+    """
+    Generate the JSON file for FRB 121102
+
+    Writes to 121102/FRB121102_host.json
+
+    All of the data currently comes from Tendulkar et al. 2017
+
+    Args:
+        build_photom (bool, optional): Generate the photometry file in the Galaxy_DB
+
+    """
+    FRB_coord = SkyCoord('05h31m58.698s +33d8m52.59s', frame='icrs')
+    # Eyeball Tendulkar+17 PA
+    gal_coord = FRB_coord.directional_offset_by(-45 * units.deg, 286e-3 * units.arcsec)
+
+    # Instantiate
+    host121102 = frbgalaxy.FRBHost(gal_coord.ra.value, gal_coord.dec.value, '121102')
+
+    # Redshift
+    host121102.set_z(0.19273, 'spec', err=0.00008)
+
+    # Photometry -- Tendulkar 2017
+    photom_file = os.path.join(db_path, 'Repeater', 'Tendulkar2017', 'tendulkar2017_photom.ascii')
+    if build_photom:
+        photom = Table()
+        #photom['Name'] = ['HG121102']  DO NOT USE str columns!
+        photom['ra'] = [host121102.coord.ra.value]
+        photom['dec'] = host121102.coord.dec.value
```

Comment:
Why this one is not a list, and the `photom['ra']` is?

Reply:
It's a quirk of `astropy.table.Table`.  To initialize the first column, you need an iterable so that it is able to read the length (even if the length is 1). Once it has the column length, it can accept a scalar value as being a single entry.